modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -123,7 +123,7 @@ mirror:
       type: ocp
     graph: true
   operators:
-  - catalog: registry.redhat.io/redhat/certified-operator-index:v4.11
+  - catalog: registry.redhat.io/redhat/certified-operator-index:v{product-version}
```

Comment:
Wonder if we should footnote here (and in the other one) that the catalog and mirror do not have to match? Might confuse customers?